server/app/views/applicant/ApplicantUpsellFragment.html

```diff
@@ -0,0 +1,29 @@
+<html th:attr="lang=${preferredLanguage.code()}">
+  <head
+    th:replace="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}"
+  ></head>
+  <body data-load-modal="true">
+    <main role="main">
+      <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
+      <h1 th:text="#{title.applicationConfirmation}"></h1>
+      <p
+        th:text="#{content.confirmed(${programName}, ${applicationId})}"
+        class="cf-application-id"
+      ></p>
+      <a href="/" class="display-flex flex-align-center">
+        <em class="usa-logo__text">
+          <button
```

Comment:
sorry I just noticed this, but you should just be able to do <a href="/" class="usa-button" ...> and not need the inner elements